src/lib/components/payload-input.svelte

```diff
@@ -1,5 +1,9 @@
 <script context="module" lang="ts">
-  export type PayloadInputEncoding = 'json/plain' | 'json/protobuf';
+  const encoding = ['json/plain', 'json/protobuf'] as const;
+  export type PayloadInputEncoding = (typeof encoding)[number];
+  export const isPayloadInputEncodingType = (
+    x: any, // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
One way to get around the `//eslint-disable` comment is to use `unknown` as a type.